webpack-intro/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "webpack-intro",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "jquery": "^3.6.1"
+  },
+  "devDependencies": {
+    "webpack": "^5.74.0"
```

Comment:
With `webpack` you need to also install the `webpack-cli` package as a dev dependency.